src/com/oltpbenchmark/benchmarks/tpcc/TPCCLoader.java

```diff
@@ -645,7 +645,7 @@ protected int loadOrders(Connection conn, int w_id, int districtsPerWarehouse, i
 			            if (order_line.ol_delivery_d != null) {
 			                orlnPrepStmt.setTimestamp(idx++, order_line.ol_delivery_d);
 			            } else {
-			                orlnPrepStmt.setNull(idx++, 0);
+							orlnPrepStmt.setTimestamp(idx++, null);
```

Comment:
Fix your formatting. This also looks to be a SpliceMachine specific change. Why doesn't "0" work?

Reply:
I kind of forget the complaining message. I just keep receiving the complaining message if I do not change it.